src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexHelpers.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Common;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.VirtualChars;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.RegularExpressions
+{
+    using RegexToken = EmbeddedSyntaxToken<RegexKind>;
+    using RegexTrivia = EmbeddedSyntaxTrivia<RegexKind>;
+
+    internal static class RegexHelpers
+    {
+        public static bool HasOption(RegexOptions options, RegexOptions val)
+            => (options & val) != 0;
+
+        public static RegexToken CreateToken(RegexKind kind, ImmutableArray<RegexTrivia> leadingTrivia, ImmutableArray<VirtualChar> virtualChars)
+            => CreateToken(kind, leadingTrivia, virtualChars, ImmutableArray<EmbeddedDiagnostic>.Empty);
+
+        public static RegexToken CreateToken(
```

Comment:
Same with this overload (it seems to only be used at line 19).

Reply:
#resolved